.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,7 @@ This pull request affects the following components: **(please check boxes)**
 
 **In raising this pull request, I confirm the following (please check boxes):**
 
-- [ ] I have read and understood the [Contributing Guidelines](/.github/CONTRIBUTING.md)?
+- [ ] I have read and understood the [Contributing Guidelines](/.github/CONTRIBUTING.md).
```

Comment:
Actually, I don't think that this link will work. While you are fixing stuff in this file, could you fix it too? :)